src/Altinn-application-owner-system/Functions/Services/Interface/IStorage.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+
+namespace AltinnApplicationOwnerSystem.Functions.Services.Interface
+{
+    /// <summary>
+    /// Interface for Storage where Application Owner system store received data
+    /// </summary>
+    public interface IStorage
+    {
+        /// <summary>
+        /// Saves a specific blob
+        /// </summary>
+        public Task SaveBlob(string name, string data);
+
+        /// <summary>
+        /// Saves a stream to bloc
+        /// </summary>
+        public Task<long> UploadFromStreamAsync(Stream stream, string fileName);
```

Comment:
To make it easier to use the interface and methods maybe be consistent in if the name of the file/blob should be the first parameter or the data itself.

Reply:
And if we by name and fileName mean the same, then consistent on that as well.